src/modules/UI/components/FlipInput/FlipInput.ui.js

```diff
@@ -120,6 +120,9 @@ export default class FlipInput extends Component<Props, State> {
         secondaryDisplayAmount: UTILS.truncateDecimals(nextProps.secondaryDisplayAmount, 2)
       })
     }
+    if (nextProps.primaryInfo.displayCurrencyCode !== this.props.primaryInfo.displayCurrencyCode) {
```

Comment:
Hmm. this will need to be added to the FlipInput2 component as well since that's the new one we're using. But it's still in PR review at this time. In a different commit, can you also add it to paul/flipinput2 in FlipInput2.ui.js and test.

Reply:
FlipInput2 has been merged to develop. Art please look at adding this change to FlipInput2 as well